### ISSUE TITLE
Ambiguous state: Participant.paid. Shold be Participant.refunded

### ISSUE THREAD

**Issue by vietlq** [makoto/blockparty#173]
Ambiguous state: Participant.paid. Shold be Participant.refunded

https://github.com/makoto/blockparty/blob/master/contracts/Conference.sol#L26

```
    struct Participant {
        string participantName;
        address addr;
        bool attended;
        bool paid;
    }
```

```
        require(participant.paid == false);

        participant.paid = true;
```

**Comment by makoto** [comment]
`refund` is true if the event is cancelled but users get deposit+extra so not sure `refund` is the correct term.

**Comment by vietlq** [comment]
`paid` can be understood 2 ways: the person paid or the person has been paid (which is used in this context).

So probably: `paid` => 'been_paid'

**Comment by makoto** [comment]
Yeah that's better. I am trying to keep interface change to minimum as it breaks the compatibilities with older contracts, but I will consider putting in when we introduce breaking changes.
